storage/buckets/create_bucket_dual_region.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package buckets
+
+// [START storage_create_bucket_dual_region]
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"cloud.google.com/go/storage"
+)
+
+// createBucketDualRegion creates a new dual-region bucket in the project in the
+// provided locations.
+func createBucketDualRegion(w io.Writer, projectID, bucketName, region1, region2 string) error {
+	// projectID := "my-project-id"
+	// bucketName := "bucket-name"
+	// region1 := "US-EAST1"
+	// region2 := "US-WEST1"
+
+	ctx := context.Background()
+
+	// Initialize the Storage client that will be used to send requests. This
```

Comment:
I don't think this comment is necessary-- we don't have it in other Go samples AFAIK.

Reply:
The comment above the client is coming from the https://googlecloudplatform.github.io/samples-style-guide/#initializing-clients.

We don't have this comment in other samples - but we should probably either update the guidance (or overrule it in the Go specific guidance) or follow it going forward?

Reply:
Sounds good.